packages/commonwealth/server/migrations/20240601170315-add-evmos.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
```

Comment:
What is the purpose of this file given: 
20221019200114-improve-chain-nodes.js assumes evmos will be in production dbs with the config in that migration.